src/Compilers/CSharp/Test/Symbol/Symbols/Source/LocalTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests;
```

Comment:
>Microsoft.CodeAnalysis.CSharp.UnitTests [](http://example.com/codeflow?start=10&length=39)

Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols?

Reply:
The other files in this directory have also this namespace, e.g.:

https://github.com/dotnet/roslyn/blob/651269a5d647d78cfe2c836da501eed693df5b0b/src/Compilers/CSharp/Test/Symbol/Symbols/Source/FieldTests.cs#L14